OracleWebLogic/dockerfiles/14.1.1.0/README.md

```diff
@@ -35,27 +35,29 @@ Before you build, select the version and distribution for which you want to buil
            -d: creates image based on 'developer' distribution
            -g: creates image based on 'generic' distribution
            -m: creates image based on 'slim' distribution
+           -j: for WebLogic 14.1.1.0 indicate Server JRE 8 or JDK 11
```

Comment:
Clarify the legal values. I can see from example below that I would pass "11" for JDK 11, but it's not clear from this message. Also, is there a default?